pkgs/tools/networking/haproxy/default.nix

```diff
@@ -1,4 +1,8 @@
-{ stdenv, pkgs, fetchurl, openssl, zlib }:
+{ useLua ? false
```

Comment:
Interesting, I haven't thought that this would be possible. But I haven't read up on scoping in nix.
So this would result into a function header looking like this:
``` nix
{ useLua ? !stdenv.isDarwin
, usePcre ? true
, stdenv, fetchurl
, openssl, zlib, lua5_3 ? null, pcre ? null
}:
# …
```
`lua5_3 ? null` and `pcre ? null` look a bit strange to me but it compiles alright.